.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 defaults: &defaults
   working_directory: ~/repo
   docker:
-    - image: circleci/node:8.9.4
+    - image: circleci/node:16
```

Comment:
Should we bump the `engine` field in `package.json`? Looks like Node 14 is in maintenance; 8 is long gone.
https://github.com/creditkarma/good-influx/blob/master/package.json#L31